auxx/src/Command/Proc.hs

```diff
@@ -372,7 +371,7 @@ createCommandProcs hasAuxxMode printAction mSendActions = rights . fix $ \comman
             <*> getArg tyEpochIndex "cur"
             <*> getArg tyBool "dry"
     , cpExec = \(i, delegatePk, curEpoch, dry) -> do
-        CmdCtx {ccPeers} <- getCmdCtx
+        CmdCtx {..} <- getCmdCtx
```

Comment:
We should change `CmdCtx` to not have `ccPeers`. Application doesn't get to choose which peers to talk to. If we need to retain this feature, we could do it in the diffusion layer: a static set of peers rather than a kademlia-backed roster.